### PR TITLE
chore(python-sdk): add documentation for account abstraction and paymaster

### PR DESCRIPTION
**Pull request by danijelTxFusion**
…on and paymaster

### CODE REVIEW

File: docs/api/python/getting-started.md [hunk 1]
```diff
@@ -566,21 +520,121 @@ if __name__ == "__main__":
...
-There is a [user guide](https://github.com/zksync-sdk/zksync2-python/blob/master/examples/README.md) on how to compile Solidity smart contracts using `zksolc` 
-compiler. `zksolc` compiler generates a `combined.json` file that contains the bytecode and ABI of a smart contract. Those files are used in the following examples. 
+With zkSync, you can deploy a contract using the create and create2 opcode, by simply building the contract into a binary format and deploying it to the zkSync network.
 
-#### Deploy a contract
+
+- [Storage](https://github.com/zksync-sdk/zksync2-examples/blob/main/solidity/storage/Storage.sol): Contract without constructor.
+- [Incrementer](https://github.com/zksync-sdk/zksync2-examples/blob/main/solidity/incrementer/Incrementer.sol): Contract with constructor.
+- [Demo](https://github.com/zksync-sdk/zksync2-examples/blob/main/solidity/demo/Demo.sol): Contract that has a dependency on 
+[Foo](https://github.com/zksync-sdk/zksync2-examples/blob/main/solidity/demo/Foo.sol) contract.
```
Comment (dutterbutter) [inline docs/api/python/getting-started.md:628]:
```suggestion
- [Foo](https://github.com/zksync-sdk/zksync2-examples/blob/main/solidity/demo/Foo.sol) contract.
```

**Comment by uF4No** [comment]
@danijelTxFusion Can you replace "zkSync" for "zkSync Era" in all places where you mention the network? For example in the getting started: "Deposit ETH from Ethereum into zkSync"

File: docs/api/python/getting-started.md [hunk 2]
```diff
@@ -1318,3 +1366,473 @@ if __name__ == "__main__":
     deploy_contract(zk_web3, account, contract_path)
 
 ```
+
+#### Deploy a smart account using crate opcode
```
Comment (uF4No) [inline docs/api/python/getting-started.md:1370]:
```suggestion
#### Deploy a smart account using create opcode
```

File: docs/api/python/getting-started.md [hunk 3]
```diff
@@ -1318,3 +1366,473 @@ if __name__ == "__main__":
...
+    # Crown token than can be minted for free
+    token_address = zk_web3.to_checksum_address("0xCd9BDa1d0FC539043D4C80103bdF4f9cb108931B")
+    constructor_arguments = {"_erc20": token_address}
+
+    # Perform contract deployment
+    deploy_account(zk_web3, account, contract_path, constructor_arguments)
+
+```
+
+#### Deploy a smart account using crate2 opcode
```
Comment (uF4No) [inline docs/api/python/getting-started.md:1509]:
```suggestion
#### Deploy a smart account using create2 opcode
```

**Review by uF4No — APPROVED** [review]

**Pull request merged**